cms/models.py

```diff
@@ -252,6 +259,16 @@ def clean(self):
             if errors:
                 raise ValidationError(errors)
 
+    def get_context(self, request, *args, **kwargs):
+        course = CoursePage.objects.get(course=self.course)
+        course_url = course.get_url() if course else ""
+        return {
+            **super().get_context(request),
+            **get_base_context(request),
+            "course_url": course_url,
+            "webinar_default_images": WEBINAR_DEFAULT_IMAGES,
+        }
+
```

Comment:
We should write a test for this as well. Won't be a frontend test but just to make sure that the required data is being  passed to the webinar details page context.